src/screens/ConfirmationScreen.js

```diff
@@ -0,0 +1,123 @@
+import {
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableOpacity,
+  Button
+} from "react-native";
+import Constants from "expo-constants";
+import * as Location from "expo-location";
+import * as TaskManager from "expo-task-manager";
+import React, { Component } from "react";
+import { LinearGradient } from "expo-linear-gradient";
+import PhoneOrLocRequest from "../components/PhoneOrLocRequest.js";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center",
+    paddingTop: Constants.statusBarHeight,
+    backgroundColor: "#ecf0f1"
+  },
+  welcome: {
+    fontSize: 20,
+    textAlign: "center",
+    margin: 10
+  },
+  instructions: {
+    textAlign: "center",
+    color: "#fff",
+    marginBottom: 5
+  },
+  paragraph: {
+    fontSize: 13,
+    textAlign: "center",
+    color: "#4bade3",
+    textAlign: "center"
+  },
+  header3: {
+    fontSize: 15,
+    fontWeight: "bold",
+    textAlign: "center",
+    margin: 10,
+    marginBottom: 30,
+    color: "#ffffff"
+  },
+  header4: {
+    fontSize: 15,
+    fontWeight: "bold",
+    textAlign: "center",
+    marginBottom: 5,
+    color: "#4bade3"
+  },
+  textBlock: {
+    width: 300,
+    padding: 10,
+    borderRadius: 5,
+    margin: 10,
+    backgroundColor: "#fff"
+  }
+});
+
+function ConfirmationScreen() {
+  return (
+    <LinearGradient colors={["#94e4f9", "#2d93d8"]} style={styles.container}>
+      <Image
+        source={require("../../CovidGuardianClient/images/simple-check.png")}
```

Comment:
I would prefer we use vector icons until we have a final design

To see the icons we have already, go to https://expo.github.io/vector-icons/ and search for what you want, any of the Ionicons are available. Examples of using the icons src\navigation\MainTabNavigator.js

If we do need to add new assets, you probably will need to load them in App.js loadResourcesAsync

Reply:
Changed images to expo icons.
The logo file is still in the images folder. I will elaborate in the notes on the image folder in this PR.